docs/aerodromes/Albury.md

```diff
@@ -27,18 +27,18 @@ For simulation purposes, visual separation is assumed to exist below the cloud b
 
 ## Coordination
 ### Departures
```

Comment:
Taxi Coordination is featured in ELW, but not AY TWR

Reply:
I'll delete all old references to taxi coord in the next pull request. There might be a couple, and I've still got to make some minor tweaks to that sort of stuff anyway. If you wanna fix it now Tyler that's fine too